src/aap_eda/settings/default.py

```diff
@@ -310,34 +329,91 @@ def _get_databases_settings() -> dict:
     "JOB_CLASS": "aap_eda.core.tasking.Job",
 }
 
+# This only takes effect if RULEBOOK_WORKER_QUEUES is not defined or
```

Comment:
Why are these variables named **RULEBOOK_** and not **ACTIVATION_**?

Reply:
To follow the new name pattern introduced by the fan out, where we are not really dealing with activations but with rulebook processes.

Reply:
we still are using "activation" in multiple places and that's fine for now because the event streams are on hold, but it is partially implemented and for the rulebook_process orchestration are processes, no longer activations. So, until perform a complete rename of everything, for now I think the new addtions should follow the new naming, not the old one.